tutorials/cassandraOutput/README.md

```diff
@@ -0,0 +1,36 @@
+# Sample Cassandra OutputOperator implementation
+
+This application generates tests data pojo and uses [CassandraPojoOputputOperator](https://github.com/apache/apex-malhar/blob/master/contrib/src/main/java/com/datatorrent/contrib/cassandra/CassandraPOJOOutputOperator.java) 
+from [Apex Malhar](https://github.com/apache/apex-malhar) library to write pojo input data to tables in cassandra database.
+
+
+Follow these steps to run this application:
+
+**Step 1**: Update/add these properties in the file `src/site/conf/properties-CassandraOutputTestApp.xml`:
+
+| Property Name  | Description |
+| -------------  | ----------- |
+| dt.operator.CassandraDataWriter.prop.store.node | cassandra server node |
+| dt.operator.CassandraDataWriter.prop.store.userName | cassandra server userName |
+| dt.operator.CassandraDataWriter.prop.store.password | cassandra server password |
+
+**Step 2**: Create database keyspace and table and add entries

```

Comment:
I had got a suggestion from someone to move this out of the application code, so adding these steps in README, this way user gets flexibility to change DB the way he wants. 
For tests we have to add ourselves, can't expect users to do configurations for test.